includes/admin/admin-actions.php

```diff
@@ -176,3 +176,35 @@ function give_hide_license_notice() {
 }
 
 add_action( 'wp_ajax_give_hide_license_notice', 'give_hide_license_notice' );
+
+
+/**
+ * Hide Outdated PHP Notice Shortly.
+ *
+ * This code is used with AJAX call to hide outdated PHP notice for a short period of time
+ *
+ * @since 1.8.9
+ *
+ * @return void
+ */
+function give_hide_outdated_php_notice() {
+
+	if ( ! isset( $_POST['_give_hide_outdated_php_notices_shortly'] ) ) {
+		die();
```

Comment:
@DevinWalker Replace `die()` with `give_die()`. Also, will make sure to use `give_die` in future work on issues.